roles/tas-install/tasks/tas.yml

```diff
@@ -0,0 +1,95 @@
+---
+- name: build TAS
+  make:   
+    target: build
```

Comment:
this seems to fail when git 1.x.x is on the host due to TAS golang deps

I think we need to add tasks in the host OS configuration layer to build and install git 2 from sources on RHEL/CentOS 7 if existing git version is lower than required by TAS